core/arch/aarch64/proc.c

```diff
@@ -192,53 +226,77 @@ proc_init_arch(void)
 #define GET_FEAT_VAL(FEATURE) (((ushort)FEATURE) & 0x000F)
 #define GET_FEAT_NSFLAG(FEATURE) ((((ushort)FEATURE) & 0x8000) >> 15)
 
+#if defined(BUILD_TESTS)
```

Comment:
I think we have to remove this #if as well.  We want proc_set_feature() to be available when using DR in standalone mode or when using the separate drdecodelib, regardless of whether tests were built.

If what we really want to do is disallow use of this API function in managed mode, we could check for `standalone_library` being set.  We could document that this is only supported in standalone mode and fail otherwise -- hmm, it has no return value.  I guess a usage assert.

Reply:
Agree, removed.